ota-plus-web/app/com/advancedtelematic/api/ApiClient.scala

```diff
@@ -48,6 +50,19 @@ object ApiRequest {
   }
 }
 
+trait CirceJsonBodyWritables {
+
+  implicit def circeJsonBodyWritable: BodyWritable[io.circe.Json] =
+    BodyWritable(json => InMemoryBody(ByteString.fromString(json.noSpaces)), "application/json")
+
+}
+
+case class CreateRepositoryRequest(keyType: KeyType)
```

Comment:
This object exists in libtuf too?

Reply:
no, though it might be put into libtuf-server.